Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "fail"
-version = "0.2.1"
+version = "0.3.0"
```

Comment:
I don't think we need to bump the version?

Reply:
It depends on whether we consider increasing the rustc version dependency a breaking change or not.